akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md

```diff
@@ -248,7 +248,7 @@ your own workstation. Docker, a JDK and @java[maven]@scala[sbt] is all that need
 1. Start a local PostgresSQL server on default port 5432. The included `docker-compose.yml` starts everything required for running locally.
 
     ```shell
-    docker-compose up -d
+    docker-compose up --wait
```

Comment:
these are still on the old `docker-compose`

Reply:
ok, confusing

Reply:
Actually, maybe on linux distros generally there can be both `docker-compose` v1 and `docker compose` v2? So probably good to update to `docker compose` everywhere.